src/api/ledger/pathfind.js

```diff
@@ -46,8 +46,12 @@ function requestPathFind(remote, pathfind: PathFind, callback) {
       _.omit(utils.common.toRippledAmount(amount), 'value'));
   }
 
-  remote.requestRipplePathFind(params,
-    composeAsync(_.partial(addParams, params), callback));
+  remote.createPathFind(params, function(data) {
+    if (data.full_reply) {
```

Comment:
Well, I think that would prevent the ability to use it for actual streaming. Here we just want to get the first full reply, but other use cases might want to continue streaming.